prisma/schema.prisma

```diff
@@ -131,6 +130,36 @@ model DeliveryLogLine {
   updatedAt DateTime    @updatedAt
 }
 
+model Conversation {
+  id        Int         @id @default(autoincrement())
+  name      String
+  createdBy Int
```

Comment:
What's the reasoning for the createdBy data field?

Reply:
We thought it would be helpful to know who is the owner of the conversation.

Reply:
If so - this should be encoded as an additional belongs-to relationship with the User model, rather than a standard Int data field